.github/workflows/test.yml

```
@@ -32,6 +32,16 @@ jobs:
       RAILS_ENV: "test"
       DATABASE_URL: "postgres://postgres:postgres@localhost:5432/forms_admin_test"
     steps:
+      - uses: nanasess/setup-chromedriver@v2
```

Let's add a comment explaining why we're doing this

```suggestion
      # TODO: remove these steps once we can use latest Chrome again (see https://github.com/teamcapybara/capybara/issues/2800)
      - uses: nanasess/setup-chromedriver@v2
```